### ISSUE TITLE
Typography - changing text color globally

### ISSUE THREAD

**Issue by DocBouCou** [quasarframework/quasar#6649]
Currently (v 1.9.10), it does not seem possible to, easily, change the default text color globally (meaning for what is reported in the "typography" section of the docs). There is no sass/stylus variable for default text color.
Now, in quasar, the text color is pure black, which is not recommended (see for instance: https://uxmovement.com/content/why-you-should-never-use-pure-black-for-text-or-backgrounds/ )
In order to have a dark grayish text, I am "bypassing" this issue by using the $dark variable, adding "text-dark" everywhere I need it, which is not convenient.
By the way, there are some questions in the forum concerning this issue : https://forum.quasar-framework.org/topic/5202/change-color-site-wise-conflicts-with-text-color and https://forum.quasar-framework.org/topic/5334/changing-default-text-color .

**Comment by francoism90** [comment]
I'm facing the same issue #6656, would you like to help me creating a PR with more CSS variables?

**Comment by pdanpdan** [comment]
You can set the color on the body or on almost any parent element and the color is propagated to the children.

**Comment by rstoenescu** [comment]
Also, Quasar does not set any default color for text, to allow customization.

**Comment by Sjoerd82** [comment]
FWIW... I just got started with Quasar, and migrating from Vuetify this was among the first issues I ran in to. But it's actually very simple to "fix" this. In src/css/app.css add:

```
// app global css in SCSS form

.body--light {
    color: rgba(0, 0, 0, 0.87)
}
  
.body--dark {
    color: rgba(0, 0, 0, 0)
}
```

These are the default Vuetify settings, the reduced black is indeed nicer on the eyes, IMHO. 
Funny, I immediatly noticed the difference of no trans vs. .87 trans when I migrated a Vuetify component to Quasar.
